packages/agent/src/canisterStatus/index.test.ts

```diff
@@ -113,6 +113,14 @@ describe('Canister Status utility', () => {
         decodeStrategy: 'raw',
       },
     ]);
+    const statusUTF8 = await getStatus([
+      {
+        kind: 'metadata',
+        path: 'candid:service',
+        key: 'candid',
+        decodeStrategy: 'hex',
```

Comment:
Shouldn't this be utf-8?
```suggestion
        decodeStrategy: 'utf-8',
```

Reply:
Yes, and i also realized that I didn't add the `utf-8` option for MetaData paths